apps/studio/components/interfaces/Storage/AnalyticsBuckets.tsx

```diff
@@ -95,15 +95,7 @@ export const AnalyticsBuckets = () => {
                     </TableRow>
                   )}
                   {analyticsBuckets.map((bucket) => (
-                    <TableRow
-                      key={bucket.id}
-                      className="cursor-pointer"
-                      onClick={(event) => {
-                        const url = `/project/${ref}/storage/analytics/buckets/${bucket.id}`
-                        if (event.metaKey) window.open(url, '_blank')
-                        else router.push(url)
```

Comment:
Removed the ability to click the whole row. Was annoying and redundant given the _View contents_ button.